tests/scriptcomp/corpus/constants.nss

```diff
@@ -1,5 +1,7 @@
 const int INT_LITERAL = 42;
-const int INT_HEX_LITERAL = 0x42;
+const int INT_HEX_LITERAL = 0x2A;
+const int INT_BINARY_LITERAL = 0b101010;
+const int INT_OCTAL_LITERAL = 0o052;
```

Comment:
I would have preferred some kind of Assert() that compares the values with the decimal to make sure they match, but that's also missing for the hex literal, so merging as-is and someone can send a new PR for this.